tests/experiments/test_trainer.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+
+import gym
+
+from tf2rl.algos.ddpg import DDPG
+from tf2rl.experiments.trainer import Trainer
+
+
+class TestTrainer(unittest.TestCase):
+    def test_empty_args(self):
```

Comment:
@ymd-h What are you testing here?

Reply:
Test whether it can accept empty dict in constructor.

In default behavior, max_steps is 1e+6 (1e+7?) and too long to execute in unit test.

Reply:
I see.